modules/apps/29-fee/types/keys.go

```diff
@@ -12,4 +14,12 @@ const (
 
 	// QuerierRoute is the querier route for IBC transfer
 	QuerierRoute = ModuleName
+
+	Version = "fee29-1"
+
+	KeyAppCapability = "app_capabilities"
 )
+
+func AppCapabilityName(portID, channelID string) string {
```

Comment:
No it's fine because there are all local names. everything is connected in a series. so multiple middleware in a stack is fine.

A middleware will claim the capability it gets using `channel capability path` and issue a capability with a local name of `app capability path`.

the name of module is implicitly included in the scoped keeper

Reply:
Will the error messages include this implicit name? If I have multiple middlewares using `app capability path`, will it be hard to debug?